src/include/distributed/merge_planner.h

```diff
@@ -2,7 +2,7 @@
  *
  * merge_planner.h
  *
- * Declarations for public functions and types related to router planning.
+ * Declarations for public functions and types related to MERGE router planning.
```

Comment:
I'd probably undo this unrelated change as this might make it harder for backporting certain changes (in the future, if needed)